mcs/tools/mdoc/Resources/stylesheet.xsl

```diff
@@ -260,6 +265,23 @@
 		</xsl:choose>
 	</xsl:template>
 
+	<xsl:template name="GetFileSystemTypeName">
+		<xsl:param name="typename" />
+		<xsl:variable name="base" select="substring-before ($typename, '&lt;')" />
```

Comment:
*Why* does this need to be moved into `GetFileSystemTypeName` from `GetEscapedTypeName`?